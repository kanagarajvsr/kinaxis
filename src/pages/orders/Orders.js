import Datatable from "../../components/datatable/Datatable";
import { Columns, Rows } from "../../jsonData/transactionData";
import "./orders.scss"; 

function Orders() {
  return (
    <div className="listContainer">
    <Datatable columns={Columns} rows={Rows} pageName="Latest Transactions" />
</div>
  )
}

export default Orders