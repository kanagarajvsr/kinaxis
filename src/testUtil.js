
import { BrowserRouter } from 'react-router-dom'
import { render,screen,fireEvent } from '@testing-library/react';
 
export const renderWithRouter = (ui, {route = '/'} = {}) => {
    window.history.pushState({}, 'Test page', route);
    const renders = render(ui, {wrapper: BrowserRouter})
    return { 
        screen,
        fireEvent,
        ...renders
    }
  }