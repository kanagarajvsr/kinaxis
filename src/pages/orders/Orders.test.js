import Orders from './Orders';
import { cleanup } from '@testing-library/react';
import {renderWithRouter} from '../../testUtil';

describe('Orders render', () => {
  afterEach(cleanup);
  const {asFragment} =  renderWithRouter(<Orders />) 
    
  test('Update Snapshot', () => { 
    expect(asFragment()).toMatchSnapshot()
  })

});
   


