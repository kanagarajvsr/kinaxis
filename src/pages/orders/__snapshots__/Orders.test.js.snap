// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Orders render Update Snapshot 1`] = `
<DocumentFragment>
  <div
    class="listContainer"
  >
    <div
      class="datatable"
    >
      <div
        class="datatableTitle"
      >
        Latest Transactions
      </div>
      <div
        aria-colcount="8"
        aria-multiselectable="false"
        aria-rowcount="5"
        class="datagrid MuiDataGrid-root MuiDataGrid-root--densityStandard css-imf0nl-MuiDataGrid-root"
        role="grid"
      >
        <div>
          <div />
        </div>
        <div
          class="MuiDataGrid-main css-204u17-MuiDataGrid-main"
        >
          <div
            class="MuiDataGrid-columnHeaders css-f3jnds-MuiDataGrid-columnHeaders"
            style="min-height: 56px; max-height: 56px; line-height: 56px;"
          >
            <div
              aria-rowindex="1"
              class="MuiDataGrid-columnHeadersInner MuiDataGrid-columnHeadersInner--scrollable css-1s0hp0k-MuiDataGrid-columnHeadersInner"
              role="row"
              style="transform: translate3d(0px, 0px, 0px);"
            >
              <div
                aria-colindex="1"
                aria-sort="none"
                class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--alignCenter MuiDataGrid-columnHeaderCheckbox"
                data-field="__check__"
                role="columnheader"
                style="width: 50px; min-width: 50px; max-width: 50px;"
                tabindex="0"
              >
                <div
                  class="MuiDataGrid-columnHeaderDraggableContainer"
                  draggable="false"
                >
                  <div
                    class="MuiDataGrid-columnHeaderTitleContainer"
                  >
                    <div
                      class="MuiDataGrid-columnHeaderTitleContainerContent"
                    >
                      <span
                        class="MuiCheckbox-root MuiCheckbox-colorPrimary MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiDataGrid-checkboxInput css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
                      >
                        <input
                          aria-label="Select all rows"
                          class="PrivateSwitchBase-input css-1m9pwf3"
                          data-indeterminate="false"
                          tabindex="-1"
                          type="checkbox"
                        />
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                          data-testid="CheckBoxOutlineBlankIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                          />
                        </svg>
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </span>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                  style="min-height: 56px; opacity: 1;"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                    data-testid="SeparatorIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M11 19V5h2v14z"
                    />
                  </svg>
                </div>
              </div>
              <div
                aria-colindex="2"
                aria-sort="none"
                class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--sortable"
                data-field="id"
                role="columnheader"
                style="width: 100px; min-width: 100px; max-width: 100px;"
                tabindex="-1"
              >
                <div
                  class="MuiDataGrid-columnHeaderDraggableContainer"
                  draggable="false"
                >
                  <div
                    class="MuiDataGrid-columnHeaderTitleContainer"
                  >
                    <div
                      class="MuiDataGrid-columnHeaderTitleContainerContent"
                    >
                      <div
                        aria-label=""
                        class="MuiDataGrid-columnHeaderTitle css-1jbbcbn-MuiDataGrid-columnHeaderTitle"
                        data-mui-internal-clone-element="true"
                      >
                        Tracking ID
                      </div>
                    </div>
                    <div
                      class="MuiDataGrid-iconButtonContainer css-ltf0zy-MuiDataGrid-iconButtonContainer"
                    >
                      <button
                        aria-label="Sort"
                        class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                        tabindex="-1"
                        title="Sort"
                        type="button"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall MuiDataGrid-sortIcon css-ptiqhd-MuiSvgIcon-root"
                          data-testid="ArrowUpwardIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                          />
                        </svg>
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </button>
                    </div>
                  </div>
                  <div
                    class="MuiDataGrid-menuIcon"
                  >
                    <button
                      aria-controls=":r4:"
                      aria-haspopup="true"
                      aria-label="Menu"
                      class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall MuiDataGrid-menuIconButton css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                      id=":r5:"
                      tabindex="-1"
                      title="Menu"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                        data-testid="TripleDotsVerticalIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                        />
                      </svg>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
                <div
                  class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                  style="min-height: 56px; opacity: 1;"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                    data-testid="SeparatorIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M11 19V5h2v14z"
                    />
                  </svg>
                </div>
              </div>
              <div
                aria-colindex="3"
                aria-sort="none"
                class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--sortable"
                data-field="product"
                role="columnheader"
                style="width: 220px; min-width: 220px; max-width: 220px;"
                tabindex="-1"
              >
                <div
                  class="MuiDataGrid-columnHeaderDraggableContainer"
                  draggable="false"
                >
                  <div
                    class="MuiDataGrid-columnHeaderTitleContainer"
                  >
                    <div
                      class="MuiDataGrid-columnHeaderTitleContainerContent"
                    >
                      <div
                        aria-label=""
                        class="MuiDataGrid-columnHeaderTitle css-1jbbcbn-MuiDataGrid-columnHeaderTitle"
                        data-mui-internal-clone-element="true"
                      >
                        Product Name
                      </div>
                    </div>
                    <div
                      class="MuiDataGrid-iconButtonContainer css-ltf0zy-MuiDataGrid-iconButtonContainer"
                    >
                      <button
                        aria-label="Sort"
                        class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                        tabindex="-1"
                        title="Sort"
                        type="button"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall MuiDataGrid-sortIcon css-ptiqhd-MuiSvgIcon-root"
                          data-testid="ArrowUpwardIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                          />
                        </svg>
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </button>
                    </div>
                  </div>
                  <div
                    class="MuiDataGrid-menuIcon"
                  >
                    <button
                      aria-controls=":r7:"
                      aria-haspopup="true"
                      aria-label="Menu"
                      class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall MuiDataGrid-menuIconButton css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                      id=":r8:"
                      tabindex="-1"
                      title="Menu"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                        data-testid="TripleDotsVerticalIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                        />
                      </svg>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
                <div
                  class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                  style="min-height: 56px; opacity: 1;"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                    data-testid="SeparatorIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M11 19V5h2v14z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            style="overflow: visible; height: 0px; width: 0px;"
          >
            <div
              class="MuiDataGrid-virtualScroller css-1w5m2wr-MuiDataGrid-virtualScroller"
              style="width: 0px; height: auto; margin-top: 56px; overflow-x: hidden;"
            >
              <div
                class="MuiDataGrid-virtualScrollerContent MuiDataGrid-virtualScrollerContent--overflowed css-1kwdphh-MuiDataGrid-virtualScrollerContent"
                style="width: auto; height: 260px; min-height: auto;"
              >
                <div
                  class="MuiDataGrid-virtualScrollerRenderZone css-s1v7zr-MuiDataGrid-virtualScrollerRenderZone"
                  style="transform: translate3d(0px, 0px, 0px);"
                >
                  <div
                    aria-rowindex="2"
                    aria-selected="false"
                    class="MuiDataGrid-row"
                    data-id="1143155"
                    data-rowindex="0"
                    role="row"
                    style="max-height: 52px; min-height: 52px;"
                  >
                    <div
                      aria-colindex="1"
                      aria-colspan="1"
                      class="MuiDataGrid-cellCheckbox MuiDataGrid-cell--withRenderer MuiDataGrid-cell MuiDataGrid-cell--textCenter"
                      data-colindex="0"
                      data-field="__check__"
                      role="cell"
                      style="min-width: 50px; max-width: 50px; min-height: 52px; max-height: 52px;"
                      tabindex="-1"
                    >
                      <span
                        class="MuiCheckbox-root MuiCheckbox-colorPrimary MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiDataGrid-checkboxInput css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
                      >
                        <input
                          aria-label="Select row"
                          class="PrivateSwitchBase-input css-1m9pwf3"
                          data-indeterminate="false"
                          tabindex="-1"
                          type="checkbox"
                        />
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                          data-testid="CheckBoxOutlineBlankIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                          />
                        </svg>
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </span>
                    </div>
                    <div
                      aria-colindex="2"
                      aria-colspan="1"
                      class="MuiDataGrid-cell MuiDataGrid-cell--textLeft"
                      data-colindex="1"
                      data-field="id"
                      role="cell"
                      style="min-width: 100px; max-width: 100px; min-height: 52px; max-height: 52px;"
                      tabindex="-1"
                    >
                      <div
                        class="MuiDataGrid-cellContent"
                      >
                        1143155
                      </div>
                    </div>
                    <div
                      aria-colindex="3"
                      aria-colspan="1"
                      class="MuiDataGrid-cell MuiDataGrid-cell--textLeft"
                      data-colindex="2"
                      data-field="product"
                      role="cell"
                      style="min-width: 220px; max-width: 220px; min-height: 52px; max-height: 52px;"
                      tabindex="-1"
                    >
                      <div
                        class="MuiDataGrid-cellContent"
                      >
                        Acer Nitro 5
                      </div>
                    </div>
                  </div>
                  <div
                    aria-rowindex="3"
                    aria-selected="false"
                    class="MuiDataGrid-row"
                    data-id="2235235"
                    data-rowindex="1"
                    role="row"
                    style="max-height: 52px; min-height: 52px;"
                  >
                    <div
                      aria-colindex="1"
                      aria-colspan="1"
                      class="MuiDataGrid-cellCheckbox MuiDataGrid-cell--withRenderer MuiDataGrid-cell MuiDataGrid-cell--textCenter"
                      data-colindex="0"
                      data-field="__check__"
                      role="cell"
                      style="min-width: 50px; max-width: 50px; min-height: 52px; max-height: 52px;"
                      tabindex="-1"
                    >
                      <span
                        class="MuiCheckbox-root MuiCheckbox-colorPrimary MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiDataGrid-checkboxInput css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
                      >
                        <input
                          aria-label="Select row"
                          class="PrivateSwitchBase-input css-1m9pwf3"
                          data-indeterminate="false"
                          tabindex="-1"
                          type="checkbox"
                        />
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                          data-testid="CheckBoxOutlineBlankIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                          />
                        </svg>
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </span>
                    </div>
                    <div
                      aria-colindex="2"
                      aria-colspan="1"
                      class="MuiDataGrid-cell MuiDataGrid-cell--textLeft"
                      data-colindex="1"
                      data-field="id"
                      role="cell"
                      style="min-width: 100px; max-width: 100px; min-height: 52px; max-height: 52px;"
                      tabindex="-1"
                    >
                      <div
                        class="MuiDataGrid-cellContent"
                      >
                        2235235
                      </div>
                    </div>
                    <div
                      aria-colindex="3"
                      aria-colspan="1"
                      class="MuiDataGrid-cell MuiDataGrid-cell--textLeft"
                      data-colindex="2"
                      data-field="product"
                      role="cell"
                      style="min-width: 220px; max-width: 220px; min-height: 52px; max-height: 52px;"
                      tabindex="-1"
                    >
                      <div
                        class="MuiDataGrid-cellContent"
                      >
                        Playstation 5
                      </div>
                    </div>
                  </div>
                  <div
                    aria-rowindex="4"
                    aria-selected="false"
                    class="MuiDataGrid-row"
                    data-id="2342353"
                    data-rowindex="2"
                    role="row"
                    style="max-height: 52px; min-height: 52px;"
                  >
                    <div
                      aria-colindex="1"
                      aria-colspan="1"
                      class="MuiDataGrid-cellCheckbox MuiDataGrid-cell--withRenderer MuiDataGrid-cell MuiDataGrid-cell--textCenter"
                      data-colindex="0"
                      data-field="__check__"
                      role="cell"
                      style="min-width: 50px; max-width: 50px; min-height: 52px; max-height: 52px;"
                      tabindex="-1"
                    >
                      <span
                        class="MuiCheckbox-root MuiCheckbox-colorPrimary MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiDataGrid-checkboxInput css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
                      >
                        <input
                          aria-label="Select row"
                          class="PrivateSwitchBase-input css-1m9pwf3"
                          data-indeterminate="false"
                          tabindex="-1"
                          type="checkbox"
                        />
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                          data-testid="CheckBoxOutlineBlankIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                          />
                        </svg>
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </span>
                    </div>
                    <div
                      aria-colindex="2"
                      aria-colspan="1"
                      class="MuiDataGrid-cell MuiDataGrid-cell--textLeft"
                      data-colindex="1"
                      data-field="id"
                      role="cell"
                      style="min-width: 100px; max-width: 100px; min-height: 52px; max-height: 52px;"
                      tabindex="-1"
                    >
                      <div
                        class="MuiDataGrid-cellContent"
                      >
                        2342353
                      </div>
                    </div>
                    <div
                      aria-colindex="3"
                      aria-colspan="1"
                      class="MuiDataGrid-cell MuiDataGrid-cell--textLeft"
                      data-colindex="2"
                      data-field="product"
                      role="cell"
                      style="min-width: 220px; max-width: 220px; min-height: 52px; max-height: 52px;"
                      tabindex="-1"
                    >
                      <div
                        class="MuiDataGrid-cellContent"
                      >
                        Redragon S101
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="Mui-resizeTriggers"
          >
            <div
              class="expand-trigger"
            >
              <div
                style="width: 1px; height: 1px;"
              />
            </div>
            <div
              class="contract-trigger"
            />
          </div>
        </div>
        <div>
          <div
            class="MuiDataGrid-footerContainer css-17jjc08-MuiDataGrid-footerContainer"
          >
            <div />
            <div
              class="MuiTablePagination-root css-rtrcn9-MuiTablePagination-root"
            >
              <div
                class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular MuiTablePagination-toolbar css-78c6dr-MuiToolbar-root-MuiTablePagination-toolbar"
              >
                <div
                  class="MuiTablePagination-spacer css-1psng7p-MuiTablePagination-spacer"
                />
                <p
                  class="MuiTablePagination-displayedRows css-levciy-MuiTablePagination-displayedRows"
                >
                  1–5 of 5
                </p>
                <div
                  class="MuiTablePagination-actions"
                >
                  <button
                    aria-label="Go to previous page"
                    class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
                    disabled=""
                    tabindex="-1"
                    title="Go to previous page"
                    type="button"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="KeyboardArrowLeftIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z"
                      />
                    </svg>
                  </button>
                  <button
                    aria-label="Go to next page"
                    class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
                    disabled=""
                    tabindex="-1"
                    title="Go to next page"
                    type="button"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="KeyboardArrowRightIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"
                      />
                    </svg>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
